src/org/zaproxy/zap/extension/spider/SpiderAPI.java

```diff
@@ -623,6 +625,14 @@ public ApiResponse handleApiView(String name, JSONObject params) throws ApiExcep
 			}
 			
 			result = resultUrls;
+		} else if (VIEW_ADDED_NODES.equals(name)) {
+			result = new ApiResponseList(name);
+			SpiderScan scan = (SpiderScan) this.getSpiderScan(params);
+			if (scan != null) {
+				for (String s : scan.getAddedNodesTableModel().getAddedNodes()) {
```

Comment:
This will lead to an exception when running in daemon mode (`getAddedNodesTableModel()` returns null).